job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/QueueServices.java

```diff
@@ -120,22 +86,24 @@ public void sendMessage(
                 getStatusCheckIntervalMillis(ScheduledExecutorConfig.getStatusCheckIntervalSeconds()),
                 statusCheckUrl, ScheduledExecutorConfig.getTrackingPipe(), workerAction.getTargetPipe());
 
-        final TaskMessage taskMessage = new TaskMessage(
+        LOG.debug("taskdata boko {}", taskDataObj);
```

Comment:
Poor log message

Reply:
Indeed, just some testing logs. I removed it